src/components/TrendingList/TrendingList.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { bindActionCreators } from "redux";
+import { connect } from 'react-redux';
+import trending from '../../actions/trending';
+import {
+  TrendingListContainerStyled,
+  ImageTrendingContainerStyled,
+} from './TrendingList.styled';
+
+class TrendingList extends React.Component {
+  constructor (props){
+    super(props);
+    props.getTrendingRequested();
+  }
+
+  render() {
+    const {
+      trending,
+    } = this.props;
+
+    const {
+      fetching,
+      ...resTrending,
+    } = trending;
+
+    return (
+      <TrendingListContainerStyled>
+        <h1>Trending Gifs</h1>
+        {
+          fetching ?
+         'Loading...' :
+          resTrending.data && resTrending.data.length > 0 ?
+            resTrending.data.map((key, idx) => (
+            <ImageTrendingContainerStyled key={key.id}>
```

Comment:
same thing: abstract this kind of situations in a stateless component outside this component